sdk/webpubsub/Microsoft.Azure.WebPubSub.Common/src/Request/ConnectEventRequest.cs

```diff
@@ -67,31 +67,31 @@ public EventErrorResponse CreateErrorResponse(WebPubSubErrorCode code, string me
             return new EventErrorResponse(code, message);
         }
 
-        internal ConnectEventRequest(
+        /// <summary>
+        /// The connect event request
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="claims"></param>
+        /// <param name="query"></param>
+        /// <param name="subprotocols"></param>
+        /// <param name="certificates"></param>
+        public ConnectEventRequest(
```

Comment:
This changes the API surface. Does this need to go through a new review now?

Reply:
Yes, @tg-msft will review the API changes.